### PR TITLE
table codelab code for first review

### PR DESCRIPTION
**Pull request by y3l2n**
Asking for a code review before I start using this to rewrite the codelabe. Will not be the final form of the code. Review is requested for JS/CSS files only

### CODE REVIEW

File: table-codelab/src/index.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+const clearTable = () => {
+  let table = document.getElementsByTagName("TABLE");
+  if (table.length > 0) {
+    document.body.removeChild(table[0]);
+  }
+}
+
+const setStyle = (data) => {
+  let fontFamily = data.theme.themeAccentFontFamily;
```
Comment (diminishedprime) [inline table-codelab/src/index.js]: use const unless the variable is being reassigned. here and throughout.

File: table-codelab/src/index.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+const dscc = require('@google/dscc');
+
+const clearTable = () => {
+  let table = document.getElementsByTagName("TABLE");
```
Comment (diminishedprime) [inline table-codelab/src/index.js]: why is `"TABLE"` in caps here? Also, why only remove the first child of the table and not the entire table?
Wouldn't `thead` be the first child so the table data would never get removed?

File: table-codelab/src/index.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+const drawViz = (data) => {
+  clearTable();
+  setStyle(data);
+
+  let table = document.createElement("table");
+  var rows = data.tables.DEFAULT.rows; 
+
+  // make the header
+  let th = document.createElement("th");
```
Comment (diminishedprime) [inline table-codelab/src/index.js]: There should probably be a `thead` element around this, even though it's not strictly required.
```
<thead>
  <tr>
    <th>header 1</th>
    <th>header 2</th>
  </tr>
</thead>
```

File: table-codelab/src/index.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
+const dscc = require('@google/dscc');
```
Comment (diminishedprime) [inline table-codelab/src/index.js]: Probably worth running prettier on this file, I see a few small inconsistencies.

File: table-codelab/src/index.js (hunk 3 above)
Comment (y3l2n) [reply table-codelab/src/index.js]: Done

File: table-codelab/src/index.js (hunk 4 above)
Comment (y3l2n) [reply table-codelab/src/index.js]: Done

**Pull request merged**